pkg/controller/jobs/leaderworkerset/leaderworkerset_webhook.go

```diff
@@ -186,10 +186,12 @@ func validateCreate(lws *LeaderWorkerSet) field.ErrorList {
 
 func validateTopologyRequest(lws *LeaderWorkerSet) field.ErrorList {
 	var allErrs field.ErrorList
+	lwsv1 := leaderworkersetv1.LeaderWorkerSet(*lws)
+	podSets, _ := lwsPodSets(&lwsv1)
```

Comment:
Please don't silence errors

Reply:
Logging is another possibility discussed, but for that we need to propagate the logger down, and in some cases it is not possible to, for example in the kubeflowjob